apis/core/lib/domain/dimension-mapping/DimensionMapping.ts

```diff
@@ -11,7 +11,7 @@ interface DictionaryEx {
   about: NamedNode
   sharedDimensions: NamedNode[]
   onlyValidTerms: boolean
-  replaceEntries(entries: KeyEntityPair[]): Map<Term, Term>
+  replaceEntries(entries: KeyEntityPair[]): boolean
```

Comment:
consider replacing `boolean`  with a more explicit type like `{ entriesChanged:  boolean }` so that client code doesn't have to guess the meaning.